pmm-app/src/pmm-add-instance/components/AddRemoteInstance/AddRemoteInstance.types.ts

```diff
@@ -1,5 +1,22 @@
 export enum TrackingOptions {
-  none = '',
+  none = 'TEST',
   pgStatements = 'qan_postgresql_pgstatements_agent',
   pgMonitor = 'qan_postgresql_pgstatmonitor_agent',
 }
+
+export interface InstanceData {
+  instanceType?: string;
+  defaultPort?: number;
+  remoteInstanceCredentials?: any;
+  discoverName?: string;
+}
+
+interface Instance {
+  type: string;
+  credentials?: any;
+}
+
+export interface AddRemoteInstanceProps {
+  instance: Instance;
+  selectInstance: (any) => void;
```

Comment:
Does this method receive any value or receives an Instance type?

Reply:
done

Reply:
Instance, thanks